src/wrappers/make-wrapper.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+var raw = require("./raw");
+var isFunction = require("../utils/is-function");
+
+function makeWrapper(command) {
+  return function (params, callback) {
+    return function (dispatch) {
+      if (callback === undefined && isFunction(params)) {
+        return dispatch(raw(command, [], params));
+      }
+      return dispatch(raw(command, params, callback));
```

Comment:
The signature is the same for all of these named wrappers though -- it's always just `namespace_methodName(params, callback)`.  The reason I like the named wrappers for internal use is because they let you declare which namespace(s) you're using when you're requiring.  If I `require('../wrappers/raw')` I could send anything at all through `raw`.